internal/core/items.rs

```diff
@@ -993,6 +993,14 @@ pub struct PropertyAnimation {
     pub easing: crate::animations::EasingCurve,
 }
 
+impl Default for PropertyAnimation {
+    fn default() -> Self {
+        // Defaults for PropertyAnimation are defined here (for internal Rust code doing programmatic animations)
+        // as well as in `builtins.slint` (for generated C++ and Rust code)
+        Self { delay: 0, duration: 0, iteration_count: 1., easing: Default::default() }
+    }
+}
```

Comment:
I did not do this, since I was told this struct should be properly initialized by the runtime to the values defined in the builtins file, so that all language bindings will see the right values automatically.

I fixed one code path were that did not happen, looks like there are some more:-)

If we do indeed require all languages to implement constructors like this, then we should make the default values in the builtin files a comment to make it very clear that these need to be managed by real code elsewhere.

But yes, functionality-wise this code is obviously correct and will fix the issue.

Reply:
There is one in `FlickableData::mouse_released`,
and i think it's silly to specify iteration_count every-time we want to start an animation from the rust code, it's much nicer to defaults it there once and for all. (ok, maybe not for all because C++...)